shared/profile/post-proof/index.stories.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import * as React from 'react'
+import {Box} from '../../common-adapters'
+import {action, storiesOf} from '../../stories/storybook'
+import Render from '.'
+
+const props = {
+  allowProofCheck: true,
+  onAllowProofCheck: () => action('onAllowProofCheck'),
```

Comment:
you don't need these closures here, `action` returns a function (same in `edit-profile` and `prove-enter-username`)